src/common/util/include/openvino/util/wstring_cast_util.hpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2018-2024 Intel Corporation
```

Comment:
```suggestion
// Copyright (C) 2018-2025 Intel Corporation
```
Check all modified files

Reply:
Done